tasks/image-segmentation/task.yaml

```diff
@@ -20,30 +24,36 @@ param_groups:
     required: false
 - name: parameters
   params:
```

Comment:
Split this parameter group into two, training parameters and validation parameters